waspc/examples/crud-testing/schema.prisma

```diff
@@ -1,5 +1,5 @@
 datasource db {
-  provider = "sqlite"
+  provider = "postgresql"
```

Comment:
Mistake that was left after the migration to `schema.prisma` file. I'm not sure how that got through.